AvaTax.TaxModule.Web/Scripts/widgets/runOrderSynchronizationWidget.js

```diff
@@ -0,0 +1,54 @@
+﻿angular.module('virtoCommerce.avataxModule')
```

Comment:
Need to little improve the current widget behavior.  Make it red when the order in out of sync state.